exporters/otlp/otlplog/otlploggrpc/internal/x/README.md

```diff
@@ -0,0 +1,36 @@
+# Experimental Features
+
+The metric SDK contains features that have not yet stabilized in the OpenTelemetry specification.
+These features are added to the OpenTelemetry Go metric SDK prior to stabilization in the specification so that users can start experimenting with them and provide feedback.
```

Comment:
```suggestion
The `otlploggrpc` exporter contains features that have not yet stabilized in the OpenTelemetry specification.
These features are added to the `otlploggrpc` exporter prior to stabilization in the specification so that users can start experimenting with them and provide feedback.
```